server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -282,11 +281,22 @@ public void fetchData() {
           if (client != null) {
             mmi = client.getManagerStats(TraceUtil.traceInfo(), context.rpcCreds());
             retry = false;
+            // Now that Manager is up, create the coordinator client
+            // to get majc information
+            Set<ServerId> managers = context.instanceOperations().getServers(ServerId.Type.MANAGER);
+            if (managers == null || managers.isEmpty()) {
+              throw new IllegalStateException(
+                  "io.getServers returned nothing for Manager, but it's up.");
+            }
+            ServerId manager = managers.iterator().next();
+            coordinatorHost = Optional.of(HostAndPort.fromString(manager.toHostPortString()));
+            getCoordinator(coordinatorHost.orElseThrow());
```

Comment:
This was a prexisting pattern, but caching the client object adds a lot complexity to the code that may not offer much benefit.  Seems like it could be removed.    Could remove the call to get the coordinator here, remove the coordinatorClient instance var, and in the single place that calls getCoordinator have it return the client in a finally block.

```suggestion
```

Reply:
Removed instance var in 8bb8598